lib/qbo_api/raise_http_exception.rb

```diff
@@ -1,32 +1,43 @@
 require 'faraday'
 require 'nokogiri'
 # @private
-module FaradayMiddleware
+module QboApi
```

Comment:
That name spacing pattern with the gem is:
```rb
class QboApi
   module Name
  ...
....
```

Reply:
🤦